mediacapture-streams/obtaining-local-multimedia-content/navigatorusermedia/deny.html

```diff
@@ -26,8 +26,13 @@ <h1 class="instructions">Description</h1>
     t.done();
   }),
   t.step_func(function (error) {
-    assert_equals(error.name, "SecurityError", "SecurityError returned");
-    assert_equals(error.constraintName, null, "constraintName attribute not set for permission denied");
+    assert_equals(error.name, "securityError", "securityError returned");
+    // NOTE ALEX: this is also nullable, but practically, all the browsers but FF set it to an empty string.
```

Comment:
The only error objects that would have a non-undefined "constraintName" would be from overconstraint event; for any other error, the correct test is to `assert_equals(error.constraintName, undefined)`